src/content/docs/agents/java-agent/heroku/java-agent-heroku.mdx

```diff
@@ -30,7 +30,7 @@ After you ensure that you [meet the requirements](#requirements), enable the New
 </Callout>
 
 1. Log in to your Heroku account.
-2. From the [APM Add-On Page](https://addons.heroku.com/newrelic), select a subscription plan.
+2. From the [APM Add-On Page](https://elements.heroku.com/addons/newrelic#planselector), select a subscription plan.
```

Comment:
@dbarnesbrownNR : Looks like this `planselector` anchor link isn't doing anything. Should this be `https://elements.heroku.com/addons/newrelic#pricing` instead? Or we could just leave off the anchor link entirely.

Reply:
Think I'm going to pull the anchor, thanks!